cli/config/configfile/file.go

```diff
@@ -43,6 +43,7 @@ type ConfigFile struct {
 	NodesFormat          string                      `json:"nodesFormat,omitempty"`
 	PruneFilters         []string                    `json:"pruneFilters,omitempty"`
 	Proxies              map[string]ProxyConfig      `json:"proxies,omitempty"`
+	BuildStreamContext   *bool                       `json:"buildStreamContext,omitempty"`
```

Comment:
I'd be good with a `build` key.

I don't think `Enable` is right, it's not enabling a feature, it's just changing the default.